galaxy_ng/app/access_control/access_policy.py

```diff
@@ -179,3 +179,7 @@ class ContainerNamespaceAccessPolicy(AccessPolicyBase):
 
 class ContainerRegistryRemoteAccessPolicy(AccessPolicyBase):
     NAME = 'ContainerRegistryRemoteViewSet'
+
+
+class RemoteSyncAccessPolicy(AccessPolicyBase):
```

Comment:
I suggest this objects to have the `Container` in the name, so we can differ easily when we are dealing with container or collection related code.

```suggestion
class ContainerRemoteSyncAccessPolicy(AccessPolicyBase):
```

Reply:
@ShaiahWren it looks like you have both the `ContainerRemoteSyncAccessPolicy` and the `RemoteSyncAccessPolicy`, I assume you can remove the latter